gensrc/proto/lake_types.proto

```diff
@@ -35,8 +34,10 @@ message DelvecCacheKeyPB {
 }
 
 message DelvecMetadataPB {
+    // Delvec file name
+    optional string delvec_file = 1;
     // from segment id to delvec page
-    map<uint32, DelvecPagePB> delvecs = 1;
+    map<uint32, DelvecPagePB> delvecs = 2;
```

Comment:
How about remove the `DelvecMetadataPB` and save `map<uint32, DelvecPagePB> delvecs` in `TabletMetadataPB` directly?